Functions/Gherkin.ps1

```diff
@@ -787,3 +787,150 @@ function ConvertTo-HashTableArray {
     }
 }
 
+function Import-FeatureFile {
```

Comment:
Should the new function be called something like New-GherkinFixture?

Reply:
Maybe. I've been thinking about this off and on since I saw the request, I feel like probably the best workflow is that you're going to want to run this repeatedly as you add test cases, so I wanted to focus on the idea of _updating_ and generating _missing_ steps, but I can't settle on a verb. Perhaps Update-GherkinFixture?

Reply:
I remember now that I settled on Import-FeatureFile because I didn't want to use the New or the Update verbs. 

The reason is I had in mind a script developer (specifically myself) being able to use a single Cmdlet in a CICD script that would both create new stub files if new feature files were created, but also create new test stubs in existing files if an analyst added new scenarios to an existing feature file. In that case it didn't seem like either the New or the Update verb was quite right. 

The idea though is that the business analysts writing scenarios, and the PowerShell developers writing the body of the tests inside the stubs, aren't the same people. So what I want is a repo where analysts contribute to the development of feature files, and then when the repo is pushed up, the CICD process automatically creates or updates the stubs as needed, and if new tests are present, they will automatically either fail or go inconclusive, and those new tests showing up in the dashboard (implemented elsewhere of course), would be one of the triggers to developers that they need to go in and fill out the tests.

So given that we want to avoid the name collision with Import, and I also think New or Update are perhaps not quite right, maybe we could use Publish instead? The alternative might be to use the New verb, but also make Update public, and New would detect if a stub already exists and just call Update in the background for you. That would make both of them available, but in reality a CICD script author could just use New all the time and update would be only for interactive use if the feature file and the gherkin test authors are the same person.

What I don't want is for the person writing the CICD pipeline script to have to go to the trouble of figuring out which files in a repo should have to have New called on them, and which ones already have some stubs written and therefore need Update called instead.